bindings/nodejs/generated/main.cc.prologue

```diff
@@ -8,5 +9,6 @@ using namespace v8;
 #include "../src/common.h"
 
 NAN_MODULE_INIT(Init) {
+  sol_init();
```

Comment:
@barbieri, I couldn't integrate the main loop by providing an implementation, because libuv does not support polling for hangup. Instead I call ```sol_run()``` from the uv main loop, but not before setting up file descriptor and timeout watches for libuv's ```uv_backend_fd()``` and ```uv_backend_timeout()```.

Thus, the outermost execution of the uv main loop hangs, because ```sol_run()``` never returns, but a re-entrant run-once-and-don't-hang call to the uv main loop happens whenever the uv backend file descriptor wakes up the watcher attached to the soletta main loop.

Additionally, I call ```sol_quit()``` whenever all soletta asynchronous callbacks have been released. This returns control to the uv main loop and stops the soletta main loop. This latter step is necessary, because node.js operates the main loop such that, when there are no more open handles, the process is expected to quit. In contrast, soletta apps are expected to call ```sol_quit()``` in order for them to terminate.

In conclusion, it is safe for me to simply call ```sol_init()``` from the bindings' entry point.